bookwyrm/connectors/openlibrary.py

```diff
@@ -235,8 +237,19 @@ def ignore_edition(edition_data):
 def get_description(description_blob):
     """descriptions can be a string or a dict"""
     if isinstance(description_blob, dict):
-        return description_blob.get("value")
-    return description_blob
+        description = markdown(description_blob.get("value"))
+    else:
+        description = markdown(description_blob)
```

Comment:
You'll want to pass the markdown parsed results into the `sanitizer.clean` function -- something like:
```
from bookwyrm.utils import sanitizer
...
sanitizer.clean(description)
```

This will remove any HTML characters that aren't on our allow-list and removes properties on elements that could cause problems (think onclick javascript events or marquee tags), and remove any malformed HTML (like unterminated tags).

Reply:
added `clean` now at the end of the function